jobs/buildhub/initialization.yml

```diff
@@ -2,6 +2,8 @@ build-hub:
   permissions:
     read:
       - system.Everyone
+    write:
+      - account:lambda
```

Comment:
Actually this is not necessary because the user running the command will become an admin automatically. If we run it with the lambda user then it is already the case.

Reply:
Agree.